cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -357,7 +365,7 @@ macro (google_cloud_cpp_enable_cleanup)
         OR (compute IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
         OR (compute_features)
         OR (experimental-bigquery_rest IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
-        OR (experimental-opentelemetry IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
+        OR (opentelemetry IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
```

Comment:
If you are still supporting (with a warning) `experimental-opentelemetry` you need to keep it here too.

Reply:
If we find `experimental-opentelemetry` in the list, we replace it with `opentelemetry`.

https://github.com/googleapis/google-cloud-cpp/blob/269a29dfb4b8131d4fa0da42cf1817df49cb7d2f/cmake/GoogleCloudCppFeatures.cmake#L324-L331